docs/source/datasets.rst

```diff
@@ -36,14 +38,42 @@ The following datasets are available:
 .. contents:: Datasets
     :local:
 
+Language Modeling
+^^^^^^^^^^^^^^^^^
+
+Language modeling datasets are subclasses of ``LanguageModelingDataset`` class.
+
+.. autoclass:: LanguageModelingDataset
+  :members: __init__
+
+
+WikiText-2
+~~~~~~~~~~
+
+.. autoclass:: WikiText2
+  :members: splits, iters
+
+
+WikiText103
+~~~~~~~~~~~
+
+.. autoclass:: WikiText103
+  :members: splits, iters
+
+
+PennTreebank
+~~~~~~~~~~~~
+
+.. autoclass:: PennTreebank
+  :members: splits, iters
 
 Sentiment Analysis
 ^^^^^^^^^^^^^^^^^^
 
 SST
 ~~~
 
-.. autoclass:: SST
+.. autoclass:: SST 
```

Comment:
nit: is there an empty space at the end? :)

Reply:
fixed